build.sbt

```diff
@@ -69,7 +69,8 @@ libraryDependencies ++= Seq(
   "org.apache.avro" % "avro" % "1.8.2" % "provided",
   "org.apache.hive" % "hive-jdbc" % "2.3.3" % "provided" excludeAll(excludeNetty, excludeNettyAll, excludeLog4j, excludeParquet),
   "org.apache.hadoop" % "hadoop-aws" % "2.7.3" % "provided",
-  "com.amazon.deequ" % "deequ" % "1.0.4" excludeAll(excludeSpark)
+  "com.amazon.deequ" % "deequ" % "1.0.4" excludeAll(excludeSpark),
+  "com.nextdoor.rollbar" % "rollbar-log4j" % "1.0.2"
```

Comment:
I think in order to use this feature
https://docs.rollbar.com/docs/java#synchronous-option
We need to create a custom appender, copy paste from here (https://github.com/zendesk/rollbar-log4j/blob/master/src/main/java/com/zendesk/rollbar/RollbarLog4jAppender.java)

Reply:
Or try fixing with a shutdown hook?

Reply:
I think the problem is they haven't implemented the close function there...